src/gpl/README.md

```diff
@@ -47,6 +47,10 @@ global_placement
     [-timing_driven_nets_percentage]
     [-pad_left pad_left]
     [-pad_right pad_right]
+    [-alpha alpha]
+    [-beta beta]
+    [-max_split_size max_split_size]
+    [-num_threads num_threads]
```

Comment:
Thread count is set at the application level. You should remove this flag and instead call `OpenRoad::getThreadCount()`.